index.bs

```diff
@@ -1611,7 +1610,7 @@ Create a named {{MLOperand}} based on a descriptor, that can be used as an input
     </div>
     1. If |name| is empty, then [=exception/throw=] a {{TypeError}}.
     1. [=Assert=]: the type of |descriptor| is {{MLOperandDescriptor}}.
-    1. [=Assert=]: If |descriptor|.{{MLOperandDescriptor/dimensions}} does not [=map/exist=], then |descriptor| defines a scalar input.
+    1. [=Assert=]: If |descriptor|.{{MLOperandDescriptor/dimensions}} does not [=map/exist=], then |descriptor| defines a scalar input, equivalent to zero dimensions.
```

Comment:
Yeah, this seems like the central point to resolve them so such that the other functions just work.

Reply:
Is the assert enough here? Should we formalize this and describe exactly what should be done (e.g. setting defaults, internal slot, etc).

Reply:
> Should we set dimensions to an empty array of unsigned long 

🤔 How should we convey that the original tensor desc's dimensions are not *actually* modified, but rather the dimensions are *effectively* set? Make a copy of the tensor desc in the pseudocode?

> Should we formalize this and describe exactly what should be done

I'm unsure if you're asking me or Ningxin, but "Allow edits by maintainers" is on if there's any particular wording you want to amend. :)

Reply:
> How should we convey that the original tensor desc's dimensions are not actually modified, but rather the dimensions are effectively set? Make a copy of the tensor desc in the pseudocode?

It should make a copy of the tensor desc. See https://github.com/webmachinelearning/webnn/issues/395

Reply:
> It should make a copy of the tensor desc. See #395

**Ningxin**: Per @inexorabletash's comment below, it sounds like a copy is already implicit `In case it isn't clear, the "make an effective copy of the tensor description" is implicit with the spec processing model`.